### ISSUE TITLE
Send the correct www-authenticate header.

### ISSUE THREAD

**Issue by Rajith90** [wso2/product-microgateway#1454]
### Description:
Microgateway send the ww-authenticate header as below

```
www-authenticate: OAuth2 realm="WSO2 API Microgateway", error="invalid token" , error_description="The access token expired"
```
The error description seems to wrong. Please refer the issue https://github.com/wso2/product-apim/issues/9439

### Steps to reproduce:
sens a request with an invalid token to microgateway

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---

**Comment by praminda** [comment]
Is this related to envoy gateway?

**Comment by Rajith90** [comment]
Fixed with #1869
